topics/icrc_29_window_post_message_transport.md

```diff
@@ -70,15 +75,14 @@ Messages are received by listening to `message` events and are considered as com
 
 Messages are sent by calling the `postMessage` method on the `establishedSource` with the `targetOrigin` parameter set to `establishedOrigin`.
 
-The window must not be automatically closed after sending a message, since the relying party could possibly send additional messages. 
-Instead, the relying party is responsible for closing the signer window.
+The window must not be automatically closed after sending a message, since the relying party could 
+possibly send additional messages over the existing communcation channel. Instead, the relying party is responsible for closing the signer window.
 
 ## Error Handling
 
 ### Unexpectedly Closed Window
 
-If the signer window is closed unexpectedly, the relying party should handle this as if it had received a `NOT_GRANTED` error.
```

Comment:
Any particular reason to have removed this reference to `NOT_GRANTED` error?

Reply:
The JSON RPC messages (either a response or an error) are meant as response to other JSON RPC messages. The communication channel being closed is not an error in response to a JSON RPC message and thus this line has been removed to resolve that.

Instead the communication channel being closed is a ICRC-29 specific transport channel error that should be handled outside the scope of the data sent through this transport channel.